recipes/transformato/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set version = "1.1.0" %}
+
+package:
+  name: transformato
+  version: 0.3
+
+source:
+  url: https://github.com/wiederm/transformato/archive/refs/tags/v0.3.tar.gz
+  sha256: 00a8b8ac33a49a179035fc2ace86c382bad0f94363d95a7aa6a0f0157c181156
+  
+build:
+  skip: true  # [win or osx]
```

Comment:
This looks like it's pure Python and could be noarch. What's driving the need to skip win/osx? If it's missing dependencies, we can definitely make this noarch and ignore the resulting CI failures.

Reply:
Thanks for the fast response! Yes it's pure python, I was just confused by the failing CI tests on win/osx I updated the recipe and added `noarch: python`